samples/benchmarks/coremark/sysbuild.cmake

```diff
@@ -8,7 +8,7 @@ include_guard(GLOBAL)
 if(NOT SB_CONFIG_IMAGE_2_BOARD STREQUAL "")
 
   ExternalZephyrProject_Add(
-    APPLICATION coremark_${SB_CONFIG_IMAGE_2_BOARD}
+    APPLICATION coremark_image_2_board
```

Comment:
Consider replacing `/` from board name with `_` in CMake to maintain naming convention. Actually term `image_X` is a bit ambiguous.

Reply:
Discussed offline. I will keep it as is since this solution is temporary anyway (see https://nordicsemi.atlassian.net/browse/NCSDK-26813)